components/SurveyModal.js

```diff
@@ -481,6 +483,27 @@ class SurveyModal extends React.Component {
     }));
   }
 
+  /*
+    When we add new fields to this survey, we'll also need to add them to
+    /static/download-survey-form-for-netlify-bots.html for Netlify Forms to recognize them.
```

Comment:
I went straight to `master` with this in https://github.com/texas-justice-initiative/website-nextjs/commit/8e7e298c960489a813af6655489d165fde8fbe0c so that we could try out the new Netlify Form in the deploy preview.